### PR TITLE
SEO-3147 PHP Desc long too short

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for PHP Desc long too short.

Regards,
Yvone.

### CODE REVIEW

File: PHP/DataManager/corsdatafetching.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-description: CORS Data Fetching
+title: CORS Data Fetching in PHP DataManager control | Syncfusion
+description: Learn here more about CORS Data Fetching in Syncfusion Essential PHP DataManager Control, its elements, and more.
 platform: php
 control: DataManager
 documentation: ug
 keywords: CORS Data Fetching, SAME-ORIGIN POLICY, Access-Control-Allow-Origin, Access-Control-Request-Method, Access-Control-Request-Headers
 ---
-# CORS Data Fetching
+# CORS Data Fetching in Essential PHP DataManager
```
Comment (Aishwarya-Ganesan) [inline PHP/DataManager/corsdatafetching.md:9]: Remove Essential word.

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.

File: PHP/DataManager/corsdatafetching.md (hunk 1 above)
Comment (Yvone-Atieno) [reply PHP/DataManager/corsdatafetching.md]: removed

File: PHP/Rating/Getting-Started.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+title: Getting started with PHP Rating control | Syncfusion
+description: Learn here more about Getting Started with Syncfusion Essential PHP Rating Control, its elements, and more.
 platform: php
 control: Rating
 documentation: ug
 keywords: Rating, rating
 ---
 
-# Getting Started
+# Getting Started with Essential PHP Rating
```
Comment (gowthamanthavasiyappan) [inline PHP/Rating/Getting-Started.md:11]: Remove Essential word.

File: PHP/Ribbon/getting-started.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+title: Getting started with PHP Ribbon control | Syncfusion	
+description: Learn here more about Rendering a basic Ribbon in Syncfusion Essential PHP Ribbon Control, its elements, and more.
 platform: php
 control: ribbon
 documentation: ug
 keywords: ejribbon, ribbon, ribbon widget, js ribbon 
 ---
 
-# Getting Started
+# Getting Started with Essential PHP Ribbon
```
Comment (gowthamanthavasiyappan) [inline PHP/Ribbon/getting-started.md:10]: Remove Essential word.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**